Assets/Scripts/Auction/AuctionDriver.cs

```diff
@@ -38,6 +38,28 @@ public class AuctionDriver : MonoBehaviour
 
     [SerializeField]
     private Animator cameraAnimator;
+    public GameObject Camera => cameraAnimator.gameObject;
+
+    public static AuctionDriver Singleton;
+
+private void Awake() {
```

Comment:
This method should be indented one level.

Reply:
Done!